### PR TITLE
Install UIKit

### PR DESCRIPTION
**Pull request by marshallds**
#### What does this pull request do?

Adds UIKit so we can do all the stories requiring styles

#### Screenshots, if applicable.
image

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,18 +1,53 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Spicoli</title>
+    <title>Radial Code Review</title>
```
Comment (fleaplus) [inline app/views/layouts/application.html.erb]: Maybe `Radial - Code Review Pro` to be consistent with the H1 below?

File: Gemfile.lock [hunk 2]
```diff
@@ -1,3 +1,9 @@
+GIT
+  remote: git://github.com/nicbet/rails-uikit.git
+  revision: 395e860901d67ffd832945a4edb6a60edc2d0304
+  specs:
+    rails-uikit (3.0.0.beta.40)
```
Comment (fleaplus) [inline Gemfile.lock]: This version of uikit is 6 months older than what is available at getuikit.com. Should we fork the gem and update it?

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -1,18 +1,53 @@
...
+                <%= image_tag '170px-Wide-Radial-Dev-Group-Logo.png', alt: 'Radial Development Group' %>
+                Code Review Pro
+              </h1>
+            </a>
+          </ul>
+        </div>
+        <div class="uk-navbar-right">
+          <ul class="uk-navbar-nav">
+            <li><a href="/users/auth/github">Get Started</a></li>
+            <li><a href="">login</a></li>
```
Comment (fleaplus) [inline app/views/layouts/application.html.erb]: capitalize login?

File: Gemfile.lock (hunk 2 above)
Comment (marshallds) [reply Gemfile.lock]: yeah, probably

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (marshallds) [reply app/views/layouts/application.html.erb]: Maybe. It gets capitalized in css tho

**Pull request merged**